### ISSUE TITLE
Review a11y

### ISSUE THREAD

**Issue by etagwerker** [phillyrb/phillyrb.org#49]
Reported by Meg Gutshall: 

> It needs to be brought up to date with A11y standards (I noticed missing alt tags and there's definitely color contrast issues with the text) and the hyperlinks need to be adjusted/deleted
